### ISSUE TITLE
Dangling Links buttons not selectable with Prism theme

### ISSUE THREAD

**Issue by claremacrae** [damiankorcz/Prism-Theme#26]
### Problem

**Actual Behaviour**

I'm finding that I cannot select the buttons at the top of the Dangling Links plugin pane as they are underneath the title:

![image](https://user-images.githubusercontent.com/4840096/148290450-0bcad174-86b2-40c2-8f62-98e2eead5fca.png)

**Expected Behaviour**

With the default style, the Dangling links pane looks like this:

![image](https://user-images.githubusercontent.com/4840096/148290328-0f1c00d9-733a-403b-b75e-5615f34fbdc3.png)

### My setup

Obsidian: .13.18
Prism: 1.2.1
OS: macOS Catalina 10.15.7

`community-plugins.json` (corrected)

```
[
  "obsidian-dangling-links",
  "obsidian-style-settings"
]
```

`core-plugins.json`

```
[
  "file-explorer",
  "sync"
]
```

`appearance.json`

```
{
  "baseFontSize": 18,
  "theme": "moonstone",
  "baseFontSizeAction": true,
  "enabledCssSnippets": [],
  "cssTheme": "Prism",
  "translucency": false
}
```

`obsidian-style-settings/data.json`

```
{
  "obsidian-prism-theme@@scroll-bar-size": 14,
  "obsidian-prism-theme@@pt-file-explorer-vertical-lines": true,
  "obsidian-prism-theme@@pt-file-explorer-text-wrap": false,
  "obsidian-prism-theme@@pt-custom-vault-banner": false,
  "obsidian-prism-theme@@nav-header-left-sidebar-position": "pt-nav-header-left-sidebar-position-top",
  "obsidian-prism-theme@@nav-header-left-sidebar-alignment": "pt-nav-header-left-sidebar-align-left",
  "obsidian-prism-theme@@nav-header-right-sidebar-alignment": "pt-nav-header-right-sidebar-align-right",
  "obsidian-prism-theme@@nav-header-right-sidebar-position": "pt-nav-header-right-sidebar-position-top"
}
```

**Comment by claremacrae** [comment]
Is there anything I can do in the settings to fix this overlap please?

**Comment by claremacrae** [comment]
Apologies - I had listed the core plugins twice, and missed out community plugins...

Now fixed

**Comment by damiankorcz** [comment]
I should have it themed for the next update. 🙂 Here's a WIP screenshot:
![image](https://user-images.githubusercontent.com/7290097/148312457-b3b16844-7297-4f65-b243-da2d8af1eb3f.png)

**Comment by claremacrae** [comment]
Super - thanks.

**Comment by damiankorcz** [comment]
This should now be addressed in Version 1.3.0.

**Comment by claremacrae** [comment]
> This should now be addressed in Version 1.3.0.

Fantastic - thanks very much indeed! :-)
